core/hoverfly_service.go

```diff
@@ -283,7 +283,16 @@ func (hf Hoverfly) GetFilteredSimulation(urlPattern string) (v2.SimulationViewV6
 		hf.version), nil
 }
 
-func (this *Hoverfly) PutSimulation(simulationView v2.SimulationViewV6) v2.SimulationImportResult {
+func (this *Hoverfly) PutSimulation(simulationView v2.SimulationViewV6, overrideExisting bool) v2.SimulationImportResult {
```

Comment:
This could break for someone using Hoverfly as a library. We should leave this public interface as it was, but created a new one for replacing simulation:
`Hoverfly.ReplaceSimulation(simulationView)`

Both funcs can call an internal one `func (this *Hoverfly) updateSimulation(simulationView v2.SimulationViewV6, overrideExisting bool) v2.SimulationImportResult ` to avoid duplicated code.

Reply:
true. fixed